app/code/Magento/Sales/Controller/AbstractController/Reorder.php

```diff
@@ -50,21 +53,25 @@ abstract class Reorder extends Action\Action implements HttpPostActionInterface
      * @param ReorderHelper|null $reorderHelper
      * @param \Magento\Sales\Model\Reorder\Reorder|null $reorder
      * @param CheckoutSession|null $checkoutSession
+     * @param StoreManagerInterface|null $storeManager
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
     public function __construct(
```

Comment:
Thank you for pointing this out, I have corrected it with reference to the URL。
Committed.

As for Guest/Reorder.php, I have reverted it back to its original state since the above fixes did not require this part to be fixed.